tests/unit/test_helper.py

```diff
@@ -36,6 +38,20 @@ def test_allow_file():
     assert not app.allowed_file("test.txt")
 
 
+def test_wrongly_upload_strategy():
+    """
+    programmers may upload strategy in a wrong way
+    """
+    prefix = os.path.join('-1/', 'strategy7')
```

Comment:
this is a local file. Why would this file exist?

Reply:
it is fine here... a bit ❕ though

Reply:
-1/strategy7 is upload path, not local path lol